.github/workflows/auto-docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Auto Docs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  generate-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python3 -m venv robot-venv
+        source robot-venv/bin/activate
+        pip install -r requirements.txt
```

Comment:
@WiktorG351 I believe only the last command is necessary

Reply:
@artur-rs adressed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/352/commits/a2bc617ebb04f86174ec5ae2e46003f1aebd3a4f#diff-ad22275c389220855fb5c424fe7960b59b03afdc63661e3dac08f4bab54c9d45L22-R23)